contrib/opportunity/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM nitlang/nit
+RUN apt-get update && apt-get install gettext libevent-dev libsqlite3-dev -y
```

Comment:
For commands, i am against putting options last as it is against most guidelines (example: for POSIX, it should not even work) and most people (including me) expects to see them before operands. For example, I first thought that you forgot the `-y` option.

See also: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/apt-get